### PR TITLE
ROE-1878 Add sanctions information to bo pages

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-1878

### Change description

Add sanctions information to input template
Updated radio buttons text according to each page and reversed the order.

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: test/controllers/beneficial.owner.gov.controller.spec.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ import {
   PAGE_TITLE_ERROR,
   SAVE_AND_CONTINUE_BUTTON_TEXT,
   SERVICE_UNAVAILABLE,
-  SHOW_INFORMATION_ON_PUBLIC_REGISTER
+  SHOW_INFORMATION_ON_PUBLIC_REGISTER, UK_SANCTIONS_DETAILS
```
Comment (mouhajer-ch) [inline test/controllers/beneficial.owner.gov.controller.spec.ts:32]: Small one, could you please add `UK_SANCTIONS_DETAILS` on a new line.
Same in the other two files.

Comment (mattch1) [reply test/controllers/beneficial.owner.gov.controller.spec.ts:32]: Moved to a new line.

File: views/includes/page-templates/beneficial-owner-individual.html [hunk 2]
```diff
@@ -45,6 +45,8 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
       {% set sanctionsListText = "Are they on the sanctions list?" %}
       {% set sanctionsListHintText = "This means that they are subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
+      {% set sanctionsListNoText = "No, they are not on the Sanctions List" %}
```
Comment (mouhajer-ch) [inline views/includes/page-templates/beneficial-owner-individual.html:48]: The `sanctionsListText` should be `Are they on the UK Sanctions List?` as for AC in your story

File: views/includes/page-templates/beneficial-owner-gov.html [hunk 3]
```diff
@@ -38,6 +38,8 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
       {% set sanctionsListText = "Is it on the sanctions list?" %}
       {% set sanctionsListHintText = "This means that it is subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
+      {% set sanctionsListNoText = "No, it is not on the Sanctions List" %}
```
Comment (mouhajer-ch) [inline views/includes/page-templates/beneficial-owner-gov.html:41]: The `sanctionsListText` should be `Is it on the UK Sanctions List?` as for AC in your story

File: views/beneficial-owner-other.html [hunk 4]
```diff
@@ -66,6 +66,8 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
         {% set sanctionsListText = "Is it on the sanctions list?" %}
         {% set sanctionsListHintText = "This means that it is subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
+        {% set sanctionsListNoText = "No, they are not on the Sanctions List" %}
```
Comment (mouhajer-ch) [inline views/beneficial-owner-other.html:69]: The `sanctionsListText` should be `Is it on the UK Sanctions List?` as for AC in your story.

Comment (mouhajer-ch) [inline views/beneficial-owner-other.html:69]: I think the `sanctionsListNoText`  should be `No, it is not on the Sanctions List`

File: views/includes/page-templates/beneficial-owner-individual.html (hunk 2 above)
Comment (mattch1) [reply views/includes/page-templates/beneficial-owner-individual.html:48]: Updated this to the precise text.

File: views/includes/page-templates/beneficial-owner-gov.html (hunk 3 above)
Comment (mattch1) [reply views/includes/page-templates/beneficial-owner-gov.html:41]: Updated this to the precise text.

File: views/beneficial-owner-other.html (hunk 4 above)
Comment (mattch1) [reply views/beneficial-owner-other.html]: Updated this to the precise text.

Comment (mattch1) [reply views/beneficial-owner-other.html]: This is not what is on the screenshot but it should be "it". Will need to put a note on the ticket for the testers about this.

**Review by mouhajer-ch — APPROVED** [review]

**Pull request merged**